ch2driver/pytpcc/runtime/loader.py

```diff
@@ -165,7 +180,15 @@ def loadWarehouse(self, w_id):
             for c_id in range(1, self.scaleParameters.customersPerDistrict+1):
                 badCredit = (c_id in selectedRows)
                 orderDate = self.computeRandomRangeDate(startOrderDate, endOrderDate)
-                c_tuples.append(self.generateCustomer(w_id, d_id, c_id, orderDate, badCredit, True))
+                c_tuple, ca_tuples, cp_tuples, cc_tuples = self.generateCustomer(w_id, d_id, c_id, orderDate, badCredit, True)
```

Comment:
This isn't directly related to the flat schema changes, but the last parameter of `generateCustomer` is not used at all and we should remove it.

(I was wondering what the last `True` argument was for, and it looks like its for `doesReplicateName` but this isn't used at all)

Reply:
Removed.